src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private final StorageDao storageDao = new StorageDaoImpl();
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user.setLogin("test@gmail.com");
+        user.setPassword("1234556");
+        user.setAge(18);
+    }
+
+    @Test
+    void register_validUser_ok() {
+        User actual = registrationService.register(user);
+        assertEquals(user, actual);
+    }
+
+    @Test
+    void register_validUserGet_ok() {
+        User register = registrationService.register(user);
+        User actual = storageDao.get(user.getLogin());
+        assertEquals(register, actual);
+    }
```

Comment:
I meant to add a test for the case if a user with the same login already saved in storage